spherov2.js-examples/src/ensemble.ts

```diff
@@ -1,16 +1,12 @@
-import { SpheroMini } from '../toys/sphero-mini';
-import { wait } from '../utils';
-import { findToys } from './lib/scanner';
-
-import { Core } from '../toys/core';
+import { Scanner, Core, SpheroMini, Utils } from 'spherov2.js-lib';
```

Comment:
The name of the lib should be `spherov2.js`

Reply:
OK, I changed the package name to `spherov2.js`.